db/migrate/20170106122400_link_documents_and_editions.rb

```diff
@@ -0,0 +1,10 @@
+class LinkDocumentsAndEditions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :content_items, :document_id, :integer
+    add_foreign_key :content_items, :documents
```

Comment:
It took 38 seconds to run on my machine, it also locks the table which I didn't realise.